visualization/dashboard/src/MLIDashboard/Controls/AxisConfigurationDialog/AxisConfigDialog.tsx

```diff
@@ -0,0 +1,295 @@
+                        )}
+                        {selectedMeta.treatAsCategorical && (
+                            <div>
+                                <Text variant={"small"} className={styles.featureText}>
+                                {`# of unique values: ${selectedMeta.sortedCategoricalValues.length}`}
+                                </Text>
+                                {this.props.canDither && (
+                                    <Checkbox 
+                                        label ={localization.AxisConfigDialog.ditherLabel}
+                                        checked={this.state.selectedColumn.options.dither}
+                                        onChange={this.ditherChecked}
+                                    />
+                                )}
+                            </div>
+                        )}
+                        {!selectedMeta.treatAsCategorical && (
+                            <div>
+                                <Text variant={"small"} className={styles.featureText}>
+                                {`Min: ${selectedMeta.featureRange.min}`} {`Max: ${selectedMeta.featureRange.max}`}
+                                </Text>
+                                {this.props.canBin && !this.props.mustBin && (
+                                    <Checkbox
+                                        label={localization.AxisConfigDialog.binLabel}
+                                        checked={this.state.selectedColumn.options.bin}
+                                        onChange={this.shouldBinClicked}
+                                    />
+                                )}
+                                {(this.props.mustBin || this.state.selectedColumn.options.bin) && this.state.binCount !== undefined && (
+                                <SpinButton
+                                    className={styles.spinButton}
+                                    labelPosition={Position.top}
+                                    label={localization.AxisConfigDialog.numOfBins}
+                                    min={this.MIN_HIST_COLS}
+                                    max={this.MAX_HIST_COLS}
+                                    value={this.state.binCount.toString()}
+                                    onIncrement={this.setNumericValue.bind(this, 1, selectedMeta)}
+                                    onDecrement={this.setNumericValue.bind(this, -1, selectedMeta)}
+                                    onValidate={this.setNumericValue.bind(this, 0, selectedMeta)}
+                                />
+                                )}
+                            </div>
+                        )}
+                    </div>)}
+                </div>
+                <PrimaryButton 
+                    text={localization.AxisConfigDialog.select}
+                    onClick={this.saveState}
+                    className ={styles.selectButton}
```

Comment:
className ={styles.selectButton}
should the space be removed?  in general, sounds like we need to add a linter to the ts code?

Reply:
linter is definitely in order. I have created an epic for engineering excellence that I'll add that to.